recipe/build.sh

```diff
@@ -20,6 +20,9 @@ CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib" \
             --with-jpeg=$PREFIX \
             --with-libtiff=$PREFIX \
             --with-png=$PREFIX \
+            --with-sqlite3=$PREFIX \
+            --with-spatialite=$PREFIX \
+            --with-curl \
```

Comment:
I just had it on a previous version of my build file, so left it over. In all my build, curl is being identified automatically, so this isn't probably needed.